torchvision/utils.py

```diff
@@ -327,6 +327,7 @@ def draw_segmentation_masks(
 def draw_keypoints(
     image: torch.Tensor,
     keypoints: torch.Tensor,
+    visibility: Optional[torch.Tensor] = None,
```

Comment:
So, it sucks, but we'll have to add this new parameter at the end of the signature instead of here (where, I agree, it would make more sense).

The reason is that user may be using purely positional arguments here, e.g. `draw_keypoints(image, keypoints, connectivity, "green", ...)` and adding a new `visibility` parameter in the middle of the signature would break that.

This is a problem something that could have been avoided by using keyword-only parameters, but we didn't 🥲